public/app/features/dashboard/panel_editor/QueriesTab.tsx

```diff
@@ -75,7 +75,8 @@ export class QueriesTab extends PureComponent<Props, State> {
   }
 
   onPanelDataUpdate(data: PanelData) {
-    console.log('got data', data);
+    // Does the *panel* type matter?  I would think only
+    // the datasource type matters...
```

Comment:
any thoughts on what we should do here?  It looks like our current approach was to check if the *panel* is angular, but I think it actually depends on if any DataSource has angular query editors.

We could cycle datasources here and check... or maybe in `onComponentUpdated` assuming the panel object changes.